torchvision/models/detection/keypoint_rcnn.py

```diff
@@ -253,10 +253,9 @@ def __init__(self, in_channels, num_keypoints):
 
     def forward(self, x):
         x = self.kps_score_lowres(x)
-        x = misc_nn_ops.interpolate(
-            x, scale_factor=float(self.up_scale), mode="bilinear", align_corners=False
+        return torch.nn.functional.interpolate(
+            x, scale_factor=float(self.up_scale), mode="bilinear", align_corners=False, recompute_scale_factor=False
```

Comment:
Isn't `up_scale` a integer (equals to 2)? As such, is there a difference between `recompute_scale_factor=True` and `False`? Or is it because the `output_size` in this case gets registered as a constant?

Reply:
With recompute_scale_factor, if input has a dim = 0, the calculated scale_factor will be 0. ONNX Runtime won't support this (throws an error if scale <= 0 in Resize op).

Just a note that I think the default value for recompute_scale_factor is changing to False soon.

Reply:
> With recompute_scale_factor, when input has a 0 dim, the inferred scale_factor will be 0.

This to me looks like a bug while inferring the scale_factor, as the batch dimension shouldn't affect the other sizes of the tensor (which are still valid).

Note that we use tensors of shape `[0, C, H ,W]`, so it is fully-possible to infer the correct scale factor I believe

Reply:
I looked more into this. Looks like this is an error in inferring scale_factor in ONNX Runtime.
Upsample_bicubic2d is exported to ONNX and running in ONNX Runtime if you have an input like: torch.randn(1, 2, 4, 6, requires_grad=True)
ONNX graph also looks ok. 
But the same mode fails for an input like torch.randn(0, 2, 4, 6, requires_grad=True)
The reason is that ONNX Runtime is inferring scale for across all dimensions:
    for (size_t i = 0, end = input_dims.size(); i < end; ++i) {
      scales[i] = static_cast<float>(output_dims[i]) / static_cast<float>(input_dims[i]);
    }
For a 4D input, output_dims is inferred output shape (e.g. [n, c, h*scale, w*scale]) and input_dims is actual input shape (e.g. [n, c, h, w]).
This is a bug in such edge cases where n = 0.
We will create an issue for ORT to address this.